test/functional/test_framework/p2p.py

```diff
@@ -539,8 +539,16 @@ def send_and_ping(self, message, timeout=60):
         self.send_message(message)
         self.sync_with_ping(timeout=timeout)
 
-    # Sync up with the node
+    def sync_send_with_ping(self, timeout=60):
+        """Ensure any to-be-received message on this connection is processed by SendMessages"""
```

Comment:
to consider:

"Ensure the recipient node has run ProcessMessages and SendMessages for all previously sent p2p messages"

Reply:
Tried to fix, thanks